.github/workflows/buildmgr.yml

```diff
@@ -204,6 +208,16 @@ jobs:
           retention-days: 1
           if-no-files-found: error
 
+      - name: Attach installer to release
```

Comment:
Is the `tools/buildmgr/cbuildgen/installer/cbuild_install.sh` OS dependent? If not, then I think you should do it in a single VM (using the `if` statement).

Reply:
It's a step of the `create_installer` job which runs only on ubuntu-20.04